models/endoscopic_tool_segmentation/configs/train.json

```diff
@@ -136,6 +137,16 @@
             "_target_": "SimpleInferer"
         },
         "handlers": [
+            {
+                "_target_": "CheckpointLoader",
+                "_disabled_": "@use_imagenet_pretrain",
+                "load_path": "/path/to/local/weight/model.pt",
```

Comment:
Can you please define the variable above and use.. instead of doing like this... with this difficult to override the load_path from command line..

@tangy5 we need this fix so that monailabel can pass override values for `load_path`

Reply:
otherwise you can mark use_imagenet_pretrain disabled by default.. and let user chose when to enable and provide the correct path

Reply:
Hi @SachidanandAlle ,
    For the second suggestion, did you mean to set the default value of `use_imagenet_pretrain` to `true`, so that the bundle could run with the imagenet pretrain weight by default? Hi @Nic-Ma , I think it would be easy to set the `use_imagenet_pretrain` parameter to `true` by default. What do you think about this?

Thanks,
Bin

Reply:
Add @KumoLiu

Reply:
Hi @binliunls, I don't think we could set the default value of `use_imagenet_pretrain` to `true` here in the model zoo since there may be some issues with the terms of use. 
Thanks!

Reply:
I suggest to define a variable like "pretrained_weights_path", it is easier to override. As for the `use_imagenet_pretrain`, I think we should make it `False` to avoid license related issues.

Hi @binliunls , another suggest is that with default settings, users must provide an existing weights, otherwise error will happen in the handler.

I think the tumor detection bundle is a good example: https://github.com/Project-MONAI/model-zoo/tree/21d85dabfb38d013d15955ad9b1556f25bbdde3c/models/pathology_tumor_detection#pretrained-weights

Maybe we can remove the handler from the config file, to ensure that if not using any pretrained weights, users can still run the bundle. What do you think?

Reply:
Hi @yiheng-wang-nv and @KumoLiu ,
     I am a little bit confused about your advices?  Now in the endoscopic tool segmentation bundle, if `use_imagenet_pretrain` is set to `true`, the bundle could run successfully with the imagenet pretrain weight, which may have some license issue. If `use_imagenet_pretrain` is set to `false`, then a pretrained weight must be provided to run the bundle. Here are two different kind of modifications I think I can do:
1, Directly update the default value of `use_image_pretrain` to `true`, so that the bundle could run directly after downloaded.
2, Keep the `use_image_pretrain` to `false`, remove the `CheckpointLoader` handler and add an explanation about how to load pretrained model with the `CheckpointLoader` .

I personally think the second one is better. What's your opinions?

Thanks,
Bin

Reply:
Hi @binliunls , I think your 2nd method is exactly what I mean : )